src/Evolution/DiscontinuousGalerkin/Observe.hpp

```diff
@@ -0,0 +1,329 @@
+// Distributed under the MIT License.
```

Comment:
I don't think this action is specific to DG, or is it? Perhaps better move it to `Evolution/Events/`. Once this is generalised to elliptic systems it should be move somewhere else, perhaps `IO/Events` (can't think of a better place right now)?